### PR TITLE
User can see list of products

### PR DESCRIPTION
**Pull request by SaraFrantzen**
[PT-link](https://www.pivotaltracker.com/story/show/174900376)
##This PR contains 
-Acceptance test user_can_see_list_of_products_spec.rb
-namespace for v1
-Generate model Product with name description and price
-Generate products controller

Sara & Facondo
Moving on with Unit test

### CODE REVIEW

File: app/controllers/api/v1/products_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Api::V1::ProductsController < ApplicationController
+	def index
+		products = Product.all
+    render json: { products: products }
+  end
+end
```
Comment (emtalen) [inline app/controllers/api/v1/products_controller.rb:6]: You need to format this method

File: spec/controllers/api/v1/products_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ProductsController, type: :controller do
+
+end
```
Comment (emtalen) [inline spec/controllers/api/v1/products_controller_spec.rb]: remove this file and make sure to check so the generators are turned off

File: spec/requests/api/v1/products/user_can_see_list_of_products_spec.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+		end
+
+		it "returns a collection of products" do
+			expect(response_json["products"].count).to eq 2
+		end
+	end
+end
+
+
+
```
Comment (emtalen) [inline spec/requests/api/v1/products/user_can_see_list_of_products_spec.rb]: The code looks good but the file needs formatting

**Review by emtalen — COMMENTED** [review]
I know you guys are not done, but the first thing you need to do it set up Semaphore for this repo. 
Other than that, just add the unit specs and amend the comments and if semaphore goes green this is good to be merged. So ping me when that is done.

File: spec/requests/api/v1/products/user_can_see_list_of_products_spec.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+RSpec.describe Api::V1::ProductsController, type: :request do
+  let!(:product1) { create(:product, name: 'Chicken Wings') }
```
Comment (oliverochman) [inline spec/requests/api/v1/products/user_can_see_list_of_products_spec.rb]: `let!(:products) { 2.times { create(:product)  }  }`

File: config/application.rb [hunk 5]
```diff
@@ -27,12 +21,21 @@ class Application < Rails::Application
     config.middleware.insert_before 0, Rack::Cors do
       allow do
         origins '*'
-        resource '*', 
-          headers: :any, 
-          methods: %i[get post put delete],
-          expose: %w(access-token expiry token-type uid client),
-          max_age: 0
+        resource '*',
+                 headers: :any,
```
Comment (oliverochman) [inline config/application.rb:25]: please correct the indentation in this file

**Review by oliverochman — CHANGES_REQUESTED** [review]

**Comment by tochman** [comment]
Please make sure to use properly formatted Markdown in the PR description. Also, when we test APIs, we use `request specs`, not `acceptance tests`. Another thing that I want to comment on is the commit messages: with one exception, you guys did a really good job! Keep it up, and remember to address the comments from whoever is reviewing your code.

**Pull request merged**